CHANGELOG.md

```diff
@@ -76,9 +76,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Allow use of same `WandbLogger` instance for multiple training loops ([#2055](https://github.com/PyTorchLightning/pytorch-lightning/pull/2055))
 
-- Fixed an issue where local variables were being collected into module_arguments ([#2048](https://github.com/PyTorchLightning/pytorch-lightning/pull/2048))
-
-- Fixed an issue with `auto_collect_arguments` collecting local variables that are not constructor arguments and not working for signatures that have the instance not named `self` ([#2048](https://github.com/PyTorchLightning/pytorch-lightning/pull/2048))
+- Fixed an issue with `_auto_collect_arguments` collecting local variables that are not constructor arguments and not working for signatures that have the instance not named `self` ([#2048](https://github.com/PyTorchLightning/pytorch-lightning/pull/2048))
```

Comment:
I suggest you guys take a look at [towncrier](https://pypi.org/project/towncrier/).
I am using it in Hydra am I am quiet happy with it.

The [changelog](https://github.com/facebookresearch/hydra/blob/master/NEWS.md) in Hydra is generated by it from news fragments added per significant PR.

Reply:
returns some errors, but Ill give it try, thx :]